libs/application-generic/src/custom-providers/index.ts

```diff
@@ -79,20 +78,11 @@ export const distributedLockService = {
       cacheInMemoryProviderService.useFactory();
 
     const service = new DistributedLockService(
-      factoryCacheInMemoryProviderService,
+      factoryCacheInMemoryProviderService
```

Comment:
Removing formatting change (side note, it's strange that a Linting error isn't thrown here, it does in my local. We should fix that separately.
```suggestion
      factoryCacheInMemoryProviderService,
```